index.bs

```diff
@@ -2021,7 +2031,31 @@ remove optional flag from the field.
 </div>
 
 <div algorithm>
-To <dfn>serialize as a list</dfn> given |iterable|, |max depth|, |child ownership|,
+To <dfn>serialize an Array-like</dfn> given |production|, |handle id|, |value|,
+|max depth|, |ownership type|, |known object|, |serialization internal map|, and |realm|:
```

Comment:
Nit: Given that only that helper requires a reference to the known object we may want to move it as well close to the special arguments.
```suggestion
To <dfn>serialize an Array-like</dfn> given |production|, |handle id|, |known object|, |value|, |max depth|, |ownership type|, |serialization internal map|, and |realm|:
```

Reply:
I'll push a follow-up PR for that proposal.